.github/workflows/post-dependabot.yml

```diff
@@ -48,6 +48,25 @@ jobs:
           git commit -m "Update NOTICE.txt"
           git push
 
+      - name: check for modified NOTICE-fips.txt
+        id: notice-fips-check
+        run: |
+          if git diff --quiet HEAD -- NOTICE-fips.txt; then
+            echo "modified=false" >> $GITHUB_OUTPUT
+          else
+            echo "modified=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: commit NOTICE-fips.txt
+        if: steps.notice-fips-check.outputs.modified == 'true'
```

Comment:
Wonder if `git diff --quiet HEAD` and `git add NOTICE*` could be used so we join the two steps in one for all the NOTICE files. WDYT?